pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -37,7 +37,7 @@ in stdenv.mkDerivation rec {
     bash-completion cairo freetype fontconfig pango
   ];
 
-  LC_ALL = "en_US.UTF-8"; # For po/make-images
+  LC_ALL = "C.UTF-8"; # For po/make-images
```

Comment:
Do we actually have to set this?

Reply:
Not sure! :innocent:

Reply:
Seemed to be fine https://github.com/NixOS/nixpkgs/pull/56390/commits/7e2fa85b62a3296205253d03a925aa3780e36f0a